gitbook-docs/urls_etc.md

```diff
@@ -69,14 +68,28 @@ If a server does not support some streaming options listed in here it must respo
 
 See [Subscription Protocol](subscription_protocol.html) for more details.
 
-##### Connection Hello
+#### Streaming Connection "Hello"
 
-Upon connection a 'hello' message is sent as follows:
+Upon accepting a connection a server MUST send a 'hello' message before any other communicaiton is sent. The format of the 'hello' message is as follows:
 
 ```json
 {
   "version": "1.1.2",
+  "source": "actisense.ngw-1.1234567",
   "timestamp": "2015-04-13T01:13:50.524Z",
   "self": "123456789"
 }
 ```
+
+"version" - The 'hello' message MUST contain a "version" element which MUST have a string value identifying the lowest version of signalk-delta that defines the format of the messages that will be sent by the server.
+
+"source" - The 'hello' message MUST contain a "source" element which MUST have a string value that identifies the server as follows: manufacturer/publisher.device/application.uuid, where:
+* maufacturer is the name of a manufacturer or publisher in the list maintained in this repository
```

Comment:
Should be no overhead to maintaining the list. Manufacturers/publishers would just add to it them selves under version control here. Documented as a JSON object it would provide a useful resource for validating messages.